packages/@o3r/storybook/project.json

```diff
@@ -22,25 +22,9 @@
       }
     },
     "compile": {
-      "executor": "@nx/js:tsc",
+      "executor": "nx:run-script",
       "options": {
-        "main": "packages/@o3r/storybook/src/public_api.ts",
-        "tsConfig": "packages/@o3r/storybook/tsconfig.build.json",
-        "outputPath": "packages/@o3r/storybook/dist",
-        "assets": [
-          "packages/@o3r/storybook/{builders,collection,migration}.json",
```

Comment:
These might be missing

Reply:
there is only `collection.json` which is already copied by `prepare:build:builders`